doc/api/cli.md

```diff
@@ -2113,6 +2129,7 @@ Node.js options that are allowed are:
 * `--experimental-import-meta-resolve`
 * `--experimental-json-modules`
 * `--experimental-loader`
+* `--experimental-noderealm`
```

Comment:
I think `test/parallel/test-process-env-allowed-flags-are-documented.js` needs to be updated.

Reply:
The test is failing in this PR (you can see at the bottom of this view). Looking closer though, I think the issue is that this list needs to be in sort order, and this new flag is in the wrong position. It should go between `--experimental-network-imports` and `--experimental-permission`.